packages/cleanup-plugin/index.ts

```diff
@@ -0,0 +1,86 @@
+import {EditorPlugin, ui, data} from '@wonderlandengine/editor-api';
+import {existsSync} from 'node:fs';
+
+/**
+ * Plugin to cleanup resources with broken links.
+ */
+export default class CleanupPlugin extends EditorPlugin {
+
+    result: Record<string, Array<string>> = {};
+
+    /* The constructor is called when your plugin is loaded */
+    constructor() {
+        super();
+        this.name = 'Project Cleanup Plugin';
+    }
+
+    /* Use this function for drawing UI */
+    draw() {
+        ui.text('Unused Resources');
+        ui.separator();
+
+        if (this.result) {
+            for (let k of Object.keys(this.result)) {
+                ui.text(`Found ${this.result[k].length.toString()} unused ${k}`);
+            }
+
+            ui.separator();
+            if (ui.button('Refresh')) {
+                this.collectResources();
+            }
+            ui.sameLine();
+            if (ui.button('Delete all')) {
+                this.cleanup();
+            }
+        } else {
+            this.collectResources();
+        }
+    }
+
+    LINK_CACHE: Record<string, boolean> = {};
+
+    /* Check whether the file linked by a resource exists, caching the result */
```

Comment:
Can you convert all docs to a typedoc thing, with params etc? We aren;t running typedoc yet I assume, but maybe in the future